src/codex_sdk/integrations/pydantic_ai_model.py

```
@@ -1299,15 +1571,17 @@ async def request_stream(
                 try:
-                    parts, usage, thread_id, _ = await self._run_codex_request(
+                    parts, usage, thread_id = await self._run_codex_request(
                         messages=messages,
-                        model_settings=model_settings,
                         model_request_parameters=model_request_parameters,
+                        hooks=hooks,
+                        resolved_thread_options=resolved_thread_options,
                         streamed=streamed,
                     )
                     streamed.finish(usage=usage, thread_id=thread_id, parts=parts)
                 except Exception as exc:  # pragma: no cover - defensive
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)